src/index.scss

```diff
@@ -5,6 +5,9 @@ The React <Content /> component uses the `main` element which IE11 doesn't recog
 
 @link https://github.com/carbon-design-system/carbon/issues/2789
 */
+
+@import 'carbon-components/scss/globals/scss/styles.scss';
+
 main {
   display: block;
 }
```

Comment:
You're missing the `feature-flags` declaration

Reply:
solved!